source/NVDAObjects/UIA/__init__.py

```diff
@@ -106,21 +172,37 @@ def _get_NVDAObjectAtStart(self):
 	def _get_bookmark(self):
 		return self.copy()
 
-	def _getControlFieldForObject(self, obj):
+	def _getControlFieldForObject(self, obj,isEmbedded=False,startOfNode=False,endOfNode=False):
+		"""
+		Fetch control field information for the given UIA NVDAObject.
+		@ param obj: the NVDAObject the control field is for.
+		@type obj: L{UIA}
+		@param isEmbedded: True if this NVDAObject is for a leaf node (has no useful children).
+		@ type isEmbedded: bool
+		@param startOfNode: True if the control field represents the very start of this object.
+		@type startOfNode: bool
+		@param endOfNode: True if the control field represents the very end of this object.
+		@type endOfNode: bool
+		@return: The control field for this object
+		@rtype: dict containing NVDA control field data.

```

Comment:
This should probably say textInfos.ControlField rather than dict. Pretty much the same thing, but might matter at some point?